v7labs/darwin-py
# V7 2297 labelbox format importer

Background
---

We are trying to add compatibility with LabelBox: 
https://docs.labelbox.com/docs/annotation-types-1

This PR includes:
 - Importation of bounding box **Images** from LabelBox into our system
 - Renamed some test files to avoid naming conflicts with pytest
 - A documentation typo fix

How can I test this
----

Just run `pytest`. 
You can check the test file, which should cover all the cases for this PR.

I would say, if you only have time for 1 thing, it would be to review the test named `test_it_imports_bboxes`, as it has the fully working example and it would be nice to have someone else verify my bbox calculations are OK.

Extra Info
----

Linear Parent ticket: https://linear.app/v7labs/issue/V7-2297/labelbox-format-importer
Linear Task ticket: https://linear.app/v7labs/issue/V7-2321/import-bounding-box-image
